docs/web/docs/1-getting_started/7-github.mdx

```diff
@@ -228,33 +239,35 @@ properties:
       version: '2012-10-17'
   role_name: iambic_github_app_lambda_execution
 ```
+2. You can then apply the template via `iambic apply resources/github/iam_role_lambda.yaml`
 
-You will need to replace the "NAME_OF_AWS_ACCOUNT_THAT_HAS_IambicHubRole" with the account name of the AWS account holding the IambicHubRole. In the event you change the secret names, you need to update those values as well.
+<!-- You will need to replace the "NAME_OF_AWS_ACCOUNT_THAT_HAS_IambicHubRole" with the account name of the AWS account holding the IambicHubRole. In the event you change the secret names, you need to update those values as well. -->
```

Comment:
Are we sure we can skip these, one of the problem is we do really need to find out the mgmt account name of the organization. for example: `aws_hub_account_name`. this is not static, it does depend on what the customer requirement is

Reply:
after call: `NAME_OF_AWS_ACCOUNT_THAT_HAS_IambicHubRole`is not a role name, is it an example of account name